### ISSUE TITLE
[ui5-middleware-stringreplacer] regression since version 0.3.7

### ISSUE THREAD

**Issue by iggycoder** [ui5-community/ui5-ecosystem-showcase#626]
Replacer does not work anymore since the refactoring in `0.3.7`. The log message is `No strings to replace provided either through the process environment or middleware config`. 

My Config:
```yaml
- name: ui5-middleware-stringreplacer
  afterMiddleware: compression
  configuration:
    debug: true
    files:
      - "**/ServiceConfig.js"
    replace:
      - placeholder: ${WEBSOCKET_PORT}
        value: 1091
```

The mentioned file `ServiceConfig.js` has some logic where I am creating a websocket url like this:
```javascript
 var sHost = (window.location.hostname === "localhost") ? window.location.hostname + ":${WEBSOCKET_PORT}" : window.location.host;
```

Worked flawlessly until version `0.3.6`.

**Comment by petermuessig** [comment]
Hi @iggycoder , the issue seems to be releated to #617  from @piejanssens - I'll check it...

**Comment by petermuessig** [comment]
Fixed with stringreplacer version `0.7.1`
